engine/function/src/templates/Basic.ftl

```diff
@@ -1688,6 +1714,14 @@ public class Basic {
         return ifelse(new BooleanVectorDirect(condition), trueCase, falseCase);
     }
 
+    /**
+     * Returns an array with null values replaced with preceding non-null value
```

Comment:
Copies the specified array, replacing elements that represent null in the Deephaven convention by the most recently encountered non-null value if one exists. Otherwise (if no such value exists), replaces those elements with Deephaven null.

[note slightly different wording: I ended with "Deephaven null" because this is an array of primitive type]